tests/test_cli_contract_help.py

```
@@ -0,0 +1,213 @@
+        "CMUX_WORKSPACE_ID",
+        "CMUX_SURFACE_ID",
+        "CMUX_TAB_ID",
+    ]:
+        env.pop(key, None)
+    no_socket = f"/tmp/cmux-no-socket-{uuid.uuid4().hex}.sock"
+    env["CMUX_SOCKET_PATH"] = no_socket
+    env["CMUX_SOCKET"] = no_socket
+    env["CMUX_CLI_SENTRY_DISABLED"] = "1"
+    env["CMUX_CLAUDE_HOOK_SENTRY_DISABLED"] = "1"
+
+    proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)